lib/formtastic/inputs/base/timeish.rb

```diff
@@ -157,7 +160,7 @@ def fragment_label_html(fragment)
         end
         
         def value
-          object.send(method) if object && object.respond_to?(method)
+          input_options[:selected] || object.send(method) if object && object.respond_to?(method)
```

Comment:
The whole line is not going to be evaluated when the object does not respond to the method. Is that expected?

Reply:
yeah, I feel like that's right. If the model doesn't have the attribute, we shouldn't be able to set the value of it.

Reply:
But this is about getting the value. If you pass the `:selected` option, it gets ignored. I don't see why it should be ignored as it is totally independent of reading the value from the model.

I believe it really should be `input_options.fetch(:selected) { object.try(method) }`.  This would be the closest to checking the `:value` in other parts of the code linked in 
https://github.com/justinfrench/formtastic/pull/1272#pullrequestreview-162524612.

Reply:
Yeah, that's true. There could be a use-case where someone wants some custom datetime field, and needs to set the value, regardless of whether the model has an attribute or not. I'll change it.